rasa/core/actions/action.py

```diff
@@ -168,6 +174,55 @@ def __str__(self) -> Text:
         return "Action('{}')".format(self.name())
 
 
+class ActionUtterPredictedResponse(Action):
+    """
+    An action which queries the Response Selector for the appropriate response.
+    """
+
+    def __init__(self, name: Text, silent_fail: Optional[bool] = False):
+        self.action_name = name
```

Comment:
This one is actually a true action name and not a template name and hence I used `action_name`

Reply:
yeah, maybe, but UtterTemplate is a proper action as well, no?

Reply:
that is because template names and action names match for UtterTemplate, here there isn't a concept of template and moreover we are querying the tracker with this action name.

Reply:
:+1: